cfn-resources/cluster/cmd/resource/resource.go

```diff
@@ -28,7 +28,7 @@ const (
 var defaultLabel = mongodbatlas.Label{Key: "Infrastructure Tool", Value: "MongoDB Atlas Terraform Provider"}
 
 var CreateRequiredFields = []string{constants.PubKey, constants.PvtKey, constants.ProjectID, constants.Name}
-var ReadRequiredFields = []string{constants.PubKey, constants.PvtKey, constants.ProjectID, constants.Name}
+var ReadRequiredFields = []string{constants.ID}
```

Comment:
we dont need the apikeys for reading the cluster?

Reply:
No, Keys are saved as part of secret that is created during creation.